src/components/main/codeView/CodeView.tsx

```diff
@@ -139,13 +141,23 @@ export default function CodeView(props: CodeViewProps) {
       return;
     }
 
-    if (focusedItem === RootNodeUid || focusedItemRef.current === focusedItem)
-      return;
+    // if (focusedItem === RootNodeUid || focusedItemRef.current === focusedItem)
+    if (focusedItem === RootNodeUid) return;
+
     if (!validNodeTree[focusedItem]) return;
 
     if (codeEditing) return;
     // Convert the indices to positions
 
+    const monacoEditor = getCurrentEditorInstance();
+    const node = validNodeTree[focusedItem];
+    const { startIndex, endIndex } = node.data as THtmlNodeData;
+
+    if (!startIndex || !endIndex || !monacoEditor) return;
+
+    const { startLineNumber, startColumn, endLineNumber, endColumn } =
```

Comment:
Do we need these startLineNumber ....... endColumn?

Reply:
we really don't need it